stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -1311,6 +1323,62 @@
       (ok { stacker: stacker,
             unlock-burn-height: new-unlock-ht }))))
 
+;; This function allows a node to register a new 'current-signer-set' during the prepare phase
+(define-private (update-current-signer-set (new-signer-set (list 4096 {signer: (buff 33)})))
+    (let 
+        (
+            (next-reward-cycle (+ u1 current-pox-reward-cycle))
+            (next-reward-cycle-burn-height (reward-cycle-to-burn-height next-reward-cycle))
+            (next-signer-set-map (map-get? reward-cycle-signing-key-ids next-reward-cycle))
+        )
+
+        ;; Assert not already set by checking for 'none' map entry
+        (asserts! (is-none next-signer-set-map) (err ERR_NEXT_SIGNER_SET_ALREADY_REGISTERED))
+
+        ;; Assert in prepare phase (in the last 100 blocks of the current cycle)
+        (asserts! 
```

Comment:
Can you break this conjunction up into two `asserts!`?  It's easier to read and test that way.